Lib/ufo2ft/outlineCompiler.py

```diff
@@ -873,6 +880,46 @@ def setupTable_post(self):
         post.minMemType1 = 0
         post.maxMemType1 = 0
 
+    def setupTable_COLR(self):
+        """
+        Compile the COLR table.
+
+        **This should not be called externally.**
+        """
+        if "COLR" not in self.tables:
+            return
+
+        from fontTools.ttLib.tables.C_O_L_R_ import LayerRecord
+        layerInfo = self.ufo.lib[COLOR_LAYERS_KEY]
+        colorLayerLists = {}
+        for baseGlyphName, layers in layerInfo.items():
+            colorLayerLists[baseGlyphName] = [
+                    LayerRecord(layerGlyphName, colorID)
+                        for layerGlyphName, colorID in layers]
+        self.otf["COLR"] = colr = newTable("COLR")
```

Comment:
@justvanrossum we can use buildCOLR function in here once we merge it https://github.com/fonttools/fonttools/pull/1826